### DISCUSSION TITLE
Frigate recordings on m.2 SSD in HA

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#13959] [Ask A Question]
I'm using the Frigate add-on in HA and have a Beelink with a second m.2 slot. I'd like to put a m.2 SSD there for dedicated frigate recording storage. 

Is that possible? Apologies if a dumb question but I can't seem to find a straight answer online & I don't want to buy it if not possible. Thanks!

**Comment by NickM-27** [comment]
this is not possible in HA OS, it is mentioned in the installation docs https://docs.frigate.video/frigate/installation#home-assistant-addon
